adoc/mgr_product_comparison.adoc

```diff
@@ -51,7 +51,7 @@ This includes features in development and features available only to their paren
 | Package Verification                                                                 | Supported              | Under Review
 | System Locking                                                                       | Supported              | Under Review
 | Configuration File Management                                                        | Supported              | Supported
-| Snapshots and Profiles                                                               | Supported              | Under Review (Profiles are supported)
+| Snapshots and Profiles                                                               | Supported              | Under Review (Profiles are supported, syncing is not)
```

Comment:
I would prefer it if we could use "synchronizing" but understand if there's no space in the table.

Reply:
We use both words in the product:

![sync](https://user-images.githubusercontent.com/250541/52323113-1138f480-29dc-11e9-9b10-f27cb27bea13.png)
![synchronization](https://user-images.githubusercontent.com/250541/52323114-11d18b00-29dc-11e9-9a90-e9cdcb9fd287.png)

(not to say that this is the right thing to do necessarily, this might as well be a bug! Not sure how the feature is described elsewhere in the documentation)

Reply:
Yeah, it's one of those ones that the old tech writer in me thinks is "correct" and I know I'm just going to have to let go of eventually. Like I said, if you need to use "sync" because of space restrictions, then go right ahead.